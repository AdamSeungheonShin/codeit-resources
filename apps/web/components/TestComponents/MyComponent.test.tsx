import * as renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer
    .create(
      <div>
        <h2>hello world</h2>
      </div>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
