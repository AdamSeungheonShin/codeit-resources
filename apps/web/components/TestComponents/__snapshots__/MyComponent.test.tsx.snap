// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2>
    hello world
  </h2>
</div>
`;
